+++ configure.ac        2019-03-15 13:23:54.636711544 +0800<br>
@@ -70,7 +70,6 @@<br>
 CFLAGS_COMMON+=" -Wformat-y2k"<br>
 CFLAGS_COMMON+=" -Wimport"<br>
 CFLAGS_COMMON+=" -Winit-self"<br>
-CFLAGS_COMMON+=" -Winline"<br>
 CFLAGS_COMMON+=" -Winvalid-pch"<br>
 CFLAGS_COMMON+=" -Wlong-long"<br>
 CFLAGS_COMMON+=" -Wmissing-field-initializers"<br>
@@ -95,7 +94,8 @@<br>
 CFLAGS_COMMON+=" -Wwrite-strings"<br>
 CFLAGS_COMMON+=" -Wno-switch-default"<br>
 CFLAGS_COMMON+=" -Wconversion"<br>
-CFLAGS_COMMON+=" -Wlong-long"<br>
+CFLAGS_COMMON+=" -Wno-long-long"<br>
+CFLAGS_COMMON+=" -Wno-implicit-fallthrough"<br>

AC_SUBST(CFLAGS_COMMON)
